### ISSUE TITLE
NERDTree disables event processing for wincmd and this lead to inconsistent behaviour of some other script

### ISSUE THREAD

**Issue by nkulikov** [scrooloose/nerdtree#261]
I've found that cursor movements executed internally by NERDTree plugin do not fire events like `WinEnter`,`BufWinEnter`,`WinLeave` so it will break some scripts like:

``` vim
" Highlight cursor only in current window
augroup CursorLine
  autocmd!
  autocmd VimEnter,WinEnter,BufWinEnter * setlocal cursorline
  autocmd WinLeave * setlocal nocursorline
augroup END
```

It can be fixed by removing `eventignore` modifications in [`function! nerdtree#exec(cmd)`](autoload/nerdtree.vim#L161) but I'm not sure that is good enough... Is there any considerable reasons for disabling event processing?

**Comment by matt-gardner** [comment]
I just ran into the same issue with the same script (trying to only highlight the current line in the active window), and after digging through the code came to the same conclusion.  I was going to post an issue here, but found this one before posting.

I tried removing the `eventignore` setting from `nerdtree#exec`, and I haven't run into any issues with it yet.  It fixed the issue I was seeing, and hasn't caused anything bad to happen that I'm aware of.  Is there some potential error that I've been lucky to avoid so far?
